File: pyxel/core/src/input.cpp
#include "pyxelcore/input.h"

namespace pyxelcore {

Input::Input() {}

Input::~Input() {}

int Input::btn(int key) {
  return 0;
}

int Input::btnp(int key, int hold, int period) {
  return 0;
}

int Input::btnr(int key) {
  return 0;
}

void Input::mouse(int visible) {}

}  // namespace pyxelcore

File: pyxelcore/src/pyxelcore/input.nim
proc mouseX(self: Pyxel): int = 0

proc mouseY(self: Pyxel): int = 0

proc mouseWheel(self: Pyxel): int = 0

proc btn(self: Pyxel, key: int): bool = false

proc btnp(self: Pyxel, key, hold, period: int): bool = false

proc btnr(self: Pyxel, key: int): bool = false

proc mouse(self: Pyxel, visible: bool) = discard
